components/common/layout/header/Motto.js

```diff
@@ -18,7 +20,7 @@ const Motto = () => {
 
   return (
     <>
-      <span className="navbar__title--screen-wide">
+      <span className={styles['navbar__title--screen--wide']}>
```

Comment:
b

Reply:
can't do it here without refactoring navbar stlyles